src/data/services/LicenseManagerApiService.js

```diff
@@ -8,6 +10,33 @@ class LicenseManagerApiService {
     const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/${subscriptionUUID}/licenses/assign/`;
     return apiClient.post(url, options, 'json');
   }
+
+  static fetchSubscriptions(options) {
+    const queryParams = {
+      ...options,
+    };
+
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/?${qs.stringify(queryParams)}`;
+    return apiClient.get(url);
+  }
+
+  static fetchSubscriptionUsers(subscriptionUUID, options) {
+    const queryParams = {
```

Comment:
Are these `queryParams` necessary if all they're doing is copying `options`? Why not just pass `options` to `qs.stringify(options)`?